invenio_app_ils/records/api.py

```diff
@@ -300,6 +310,14 @@ class Item(_Item):
         "SCANNING",
     ]
     CIRCULATION_RESTRICTIONS = ["NO_RESTRICTION"]
```

Comment:
Since you have touched this, may I ask you to implement an extra functionality?
We need to have more value for `CIRCULATION_RESTRICTIONS`, we can already add them as default in ILS. We currently have: 4 weeks, 1 week, reference

Could you please:
* set `CIRCULATION_RESTRICTIONS = ["NO_RESTRICTION", "1 week", "4 weeks"]` Do you think it makes sense to add also 2 and 3 weeks?
* adjust the ui item form to have one this value selectable when changing an item
* bonus (I don't know if it makes sense in this PR): in `circulation.utils` there are few methods that need to be re-implemented taking into account this new restrictions (to be checked which ones exactly).
* just food for thoughts: shall we someone block changes on the `CIRCULATION_RESTRICTIONS` if the item is currently on loan? Or would it be possible to simply show a warning in the edit form, e.g. "You are editing an item that is currently on loan (link to the loan)"?

Reply:
* I think that's something the library could answer the best but it seems better to add 2 and 3 weeks too in case there is a need for it. But should we keep the naming convention consistent and call them `"1_WEEK"` or `"ONE_WEEK"`?
* What do you mean by "selectable"?
* I'd prefer to do it in another PR especially since we'd need to go over them and discuss
* We have an issue open to go over the validation (https://github.com/inveniosoftware/invenio-app-ils/issues/569) and I added your comment to the issue.

Reply:
Right now the labels are defined together with the value in `invenioConfig` (e.g. https://github.com/inveniosoftware/invenio-app-ils/pull/570/files#diff-72fd3b444706594e8f08391f899add9eR151). Can we store this information in `invenioConfig` or would it make more sense to have a `uiConfig` or similar?

Are we gonna do the UX updates for forms as we make them now or is the plan to go over them all later?